src/GeekPlatform/Views/Kurzusaim/Index.cshtml

```diff
@@ -1,50 +1,76 @@
 ﻿@{
     ViewData["Title"] = "Kurzusaim";
 }
+@using GeekPlatform.ViewModels.Kurzusaim;
+
 <div class="row">
     <div class="col-md-12">
         <div class="my-courses-text">Kurzusaim</div>
     </div>
 </div>
-<div class="row">
-    <div class="col-md-12">
-        <div class="panel panel-default panel-course panel-active-course">
-            <div class="panel-heading panel-course-header panel-active-course-header">
-                <span class="active-course-header-text">Aktív kurzus neve</span>
-                <i class="material-icons md-36">school</i>
-            </div>
-            <div class="panel-body panel-course-body panel-active-course-body">
-                <div class="row">
-                    <div class="col-xs-5 panel-course-col panel-course-col-right-border">
-                        <div class="panel-course-col-header">Aktuális</div>
-                        <div class="panel-course-col-text">Aktuális témakör</div>
-                    </div>
-                    <div class="col-xs-5 panel-course-col panel-course-col-right-border">
-                        <div class="panel-course-col-header">Következő óra</div>
-                        <div class="panel-course-col-text">Következő téma</div>
-                        <div class="panel-course-col-next-class">YYYY. MM. DD.</div>
-                    </div>
-                    <div class="col-xs-2 panel-course-col panel-course-col-no-border">
-                        <i class="material-icons md-36 icon-blue"><a asp-controller="KurzusAdatok" asp-action="Passziv">home</a></i>
+
+@foreach (KurzusViewModel aktivKurzus in Model.AktivKurzusok)
+{
+    <div class="row">
+        <div class="col-md-12">
+            <div class="panel panel-default panel-course panel-active-course">
+                <div class="panel-heading panel-course-header panel-active-course-header">
+                    <span class="active-course-header-text">@aktivKurzus.Nev</span>
+                    <i class="material-icons md-36">school</i>
+                </div>
+                <div class="panel-body panel-course-body panel-active-course-body">
+                    <div class="row">
+                        <div class="col-xs-5 panel-course-col panel-course-col-right-border">
+                            <div class="panel-course-col-header">Aktuális</div>
+                            @if (aktivKurzus.AktualisTematika != null)
+                            {
+
+                                <div class="panel-course-col-text">@aktivKurzus.AktualisTematika.Temakor</div>
```

Comment:
Nekem itt egy UI-s megjegyzésem lenne: írjuk ki ennél is az alkalom dátumát, mint a következő óránál van - véletlenül maradhatott le a grafika tervről